### PR TITLE
[shopsys] github token erase

### LINKED ISSUE #518: Remove github auth token
### What is happening
After cleaning `composer.json` from custom repositories, `composer install` speeded up rapidly due to not checking half of the internet. This also means that `composer install` performs much less request to github, and it seems that github doesn't require auth token anymore.

### Expected result
Check if the `composer install` doesn't require auth token anymore. If so, remove it from `docker-compose.yml` templates, `Dockerfile`(s), documentation, `.travis` configs, build processes on heimdall and travis... just from everywhere as it is no longer needed

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| #518 
|New feature| No 
|BC breaks| Yes 
|Fixes issues| closes #518 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ There is a list of all the repositories maintained by monorepo, changes in log b
         - run `docker-sync start` to create volumes
     - run `docker-compose up -d --build --force-recreate` to start application
     - phing target for checking and fixing standards has changed, update `build-dev.xml` according to the changes
+- [#551 - github token erase](https://github.com/shopsys/shopsys/pull/551)
```
Comment (PetrHeinz) [inline UPGRADE.md]: This upgrade note should be optional
```suggestion
- *(optional)* [#551 - github token erase](https://github.com/shopsys/shopsys/pull/551)
```

File: UPGRADE.md [hunk 2]
```diff
@@ -81,6 +81,9 @@ There is a list of all the repositories maintained by monorepo, changes in log b
...
+    - update all `docker-compose.yml` config files and all dockerfile configs according to the changes
```
Comment (PetrHeinz) [inline UPGRADE.md]: You can be more specific than to force devs to look at the original PR:
```suggestion
    - remove the lines mentioning `github_oauth_token` from your `docker/php-fpm/Dockerfile` and `docker-compose.yml`
```

File: docs/installation/installation-using-docker-on-production-server.md [hunk 3]
```diff
@@ -224,7 +224,6 @@ docker build \
     --target production \
     -t production-php-fpm \
     --compress \
-    --build-arg github_oauth_token=PERSONAL_ACCESS_TOKEN_FROM_GITHUB \
     .
 ```
 Replace the `PERSONAL_ACCESS_TOKEN_FROM_GITHUB` string by the token generated on [Github -> Settings -> Developer Settings -> Personal access tokens](https://github.com/settings/tokens/new?scopes=repo&description=Composer+API+token).
```
Comment (PetrHeinz) [inline docs/installation/installation-using-docker-on-production-server.md]: You've missed this one :slightly_smiling_face: 
```suggestion
```

File: UPGRADE.md [hunk 4]
```diff
@@ -81,6 +81,9 @@ There is a list of all the repositories maintained by monorepo, changes in log b
...
+    - rebuild `php-fpm` container
 
 ### [shopsys/shopsys]
```
Comment (PetrHeinz) [inline UPGRADE.md]: There should be an optional upgrade note for the monorepo as well:

```markdown
- *(optional)* [#551 - github token erase](https://github.com/shopsys/shopsys/pull/551)
    - you can stop providing the `github_oauth_token` in your `docker-compose.yml`
```

**Comment by boris-brtan** [comment]
THX, fixup added

**Comment by boris-brtan** [comment]
rebased again after #566 was merged

**Pull request merged**